scripts/ecs/appsignals/sample-app/task-definitions/spring-petclinic-billing-service.json

```diff
@@ -0,0 +1,182 @@
+      "essential": true,
+      "environment": [
+        {
+          "name": "DJANGO_SETTINGS_MODULE",
+          "value": "pet_clinic_billing_service.settings"
+        },
+        {
+          "name": "PYTHONPATH",
+          "value": "/otel-auto-instrumentation-python/opentelemetry/instrumentation/auto_instrumentation:/app:/otel-auto-instrumentation-python"
+        },
+        {
+          "name": "OTEL_EXPORTER_OTLP_PROTOCOL",
+          "value": "http/protobuf"
+        },
+        {
+          "name": "OTEL_TRACES_SAMPLER_ARG",
+          "value": "endpoint=http://localhost:2000"
+        },
+        {
+          "name": "OTEL_LOGS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_PYTHON_CONFIGURATOR",
+          "value": "aws_configurator"
+        },
+        {
+          "name": "OTEL_TRACES_SAMPLER",
+          "value": "xray"
+        },
+        {
+          "name": "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+          "value": "http://localhost:4316/v1/traces"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_EXPORTER_ENDPOINT",
+          "value": "http://localhost:4316/v1/metrics"
+        },
+        {
+          "name": "OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "OTEL_RESOURCE_ATTRIBUTES",
+          "value": "service.name=spring-petclinic-demo-billing-service"
+        },
+        {
+          "name": "OTEL_METRICS_EXPORTER",
+          "value": "none"
+        },
+        {
+          "name": "OTEL_PYTHON_DISTRO",
+          "value": "aws_distro"
+        },
+        {
+          "name": "EUREKA_SERVER_URL",
+          "value": "discovery-server-cluster-name.ecs-petclinic"
+        },
+        {
+          "name": "BILLING_SERVICE_IP",
+          "value": "billing-service-cluster-name.ecs-petclinic"
+        },
+        {
+          "name": "DB_NAME",
+          "value": "postgres"
+        },
+        {
+          "name": "DB_USER",
+          "value": "djangouser"
+        },
+        {
+          "name": "DB_USER_PASSWORD",
+          "value": "asdfqwer"
+        },
+        {
+          "name": "DATABASE_PROFILE",
+          "value": "postgresql"
+        },
+        {
+          "name": "DB_SERVICE_HOST",
+          "value": "db_service_host"
+        },
+        {
+          "name": "DB_SERVICE_PORT",
+          "value": "5432"
+        }
+      ],
+      "command": ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8800 --noreload"],
+      "portMappings": [
+        {
+          "containerPort": 8800,
+          "protocol": "tcp"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/pet-clinic-billing-service",
+          "awslogs-create-group": "true",
+          "awslogs-region": "us-west-2",
+          "awslogs-stream-prefix": "ecs"
+        }
+      },
+      "healthCheck": {
```

Comment:
Why do we need health check?

Reply:
Was using it to make sure the service is keep running succeed when doing local test. Removed.